docs/network-flow-visibility.md

```diff
@@ -43,6 +47,51 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+### Prerequisites
+
+To get started, make sure you have installed Antrea with the Flow Exporter
+feature enabled by setting the following `antrea-agent` config parameter in
+the Antrea deployment manifest.
+
+```yaml
+  antrea-agent.conf: |
+    ...
+    featureGates:
+      ...
+      FlowExporter: true
+```
+
+### Theia Installation
+
+To enable both Grafana Flow Collector and
+[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please clone
+the theia repository, and install Theia and Flow Aggregator by runnning the
+following commands:
+
+```bash
+git clone https://github.com/antrea-io/theia.git
+helm install theia theia/build/charts/theia --set sparkOperator.enable=true -n flow-visibility --create-namespace
+helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
+```
+
+To enable only Grafana Flow Collector, please clone the theia repository,
+and install Theia and Flow Aggregator by runnning the following commands:
+
+```bash
+git clone https://github.com/antrea-io/theia.git
+helm install theia theia/build/charts/theia -n flow-visibility --create-namespace
+helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true -n flow-aggregator --create-namespace
```

Comment:
I think the order to theia and flow-aggregator installation is not relevant, but perhaps from a conceptual perspective would it better to list flow-aggregator before theia?

Reply:
The order is changed, thanks